articles/stock-price-prediction-using-python/index.md

```diff
@@ -0,0 +1,170 @@
+# stocks-prices-predection-using-python
+According to [wikipedia](https://en.wikipedia.org/wiki/Artificial_neural_network#:~:text=An%20artificial%20neural%20network%20is,to%20the%20input%20of%20another.) an artificial neural network is an interconnected group of nodes, inspired by a simplification of neurons in a brain. Here, each circular node represents an artificial neuron and an arrow represents a connection from the output of one artificial neuron to the input of another.
+
+In this tutorial we are going to build an ai neural network model to predict the stock price. Specifically **Tesla** stock, hope that we can make Elon Mask happy along the way.
+
+### Prerequisites
+- A good understanding of Python programming language.
+- A good understanding of neural networks, you can start from [here](https://www.section.io/engineering-education/introduction-to-neural-networks/).
+
+#### Development Environment
+- Locally install Python and Tensorflow.
+- Google Colab [Recommended].

```

Comment:
Share the link to your notebook with all the code and output in it.

Reply:
https://colab.research.google.com/github/ahmadmardeni1/Stock-price-predection-using-Python/blob/main/stock_price_predection.ipynb#scrollTo=QI4PhD_bi9dN